pkgs/servers/code-server/default.nix

```diff
@@ -12,17 +12,31 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "code-server";
-  version = "3.4.1";
-  commit = "d3773c11f147bdd7a4f5acfefdee23c26f069e76";
+  version = "3.6.0";
```

Comment:
and line 21: The version number probably shouldn't be define in two places. I think we can safely move it in the let.

Reply:
Thanks for the catch! Updated to reference the derivation's version, let me know if you had something else in mind